docs/src/documentation-pages/dev/quickstart.md

```diff
@@ -7,27 +7,29 @@ This quickstart describes the initial setup required to run an instance of Cross
 
 ### Initial Setup
 
-1. Install [Node.js](https://nodejs.org/en/download/) 18 and [Docker Compose](https://docs.docker.com/compose/install/). Make sure the Docker daemon is running.
+1. Mac Users - before starting the initial setup, ensure you have already completed the following: [development environment for mac-based computers](https://github.com/cisagov/development-guide/blob/develop/dev_envs/mac-env-setup.md).
```

Comment:
Recommend adding note for M1/M2 Mac Users as they might need extra configurations/settings when dealing with certain dependancies Crossfeed dependancies that are not mentioned on CISA development guide.